src/main/scala/akka/persistence/cassandra/compaction/SizeTieredCompactionStrategy.scala

```diff
@@ -15,12 +15,11 @@ class SizeTieredCompactionStrategy(config: Config) extends BaseCompactionStrateg
       .asScala
       .map(_.getKey)
       .forall(SizeTieredCompactionStrategy.propertyKeys.contains(_)),
-    s"Config contains properties not supported by a ${SizeTieredCompactionStrategy.ClassName}"
-  )
+    s"Config contains properties not supported by a ${SizeTieredCompactionStrategy.ClassName}")
 
   val bucketHigh: Double = if (config.hasPath("bucket_high")) config.getDouble("bucket_high") else 1.5
   val bucketLow: Double = if (config.hasPath("bucket_low")) config.getDouble("bucket_low") else 0.5
-  val coldReadsToOmit: Double = if (config.hasPath("cold_reads_to_omit")) config.getDouble("cold_reads_to_omit") else 0.05
+  // FIXME val coldReadsToOmit: Double = if (config.hasPath("cold_reads_to_omit")) config.getDouble("cold_reads_to_omit") else 0.05
```

Comment:
In Cassandra 2.1.5, cold_reads_to_omit was depreciated mostly, but retained in the code, but SizeTieredCompactionStrategyOptions.java just removed it. Furthermore, in 2.2, it ignored it, but in 3.0, any unremoved foreign properties results in an error for the CQL: https://github.com/apache/cassandra/blob/cassandra-3.0/src/java/org/apache/cassandra/schema/CompactionParams.java#L122.